spec/models/like_spec.rb

```diff
@@ -1,5 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'associations' do
+    it { should belong_to(:author).class_name('User') }
+    it { should belong_to(:post) }
+  end
+
+  describe 'update_post_likes_counter' do
+    let!(:user) { User.create!(name: 'John Doe', email: 'jhone@doe.com', password: '123456', post_counter: 0) }
+    let!(:post) { Post.create!(title: 'My First Post', author: user, comments_counter: 0, likes_counter: 0) }
+    let!(:like) { Like.new(author: user, post:) }
+
+    it 'increments the post likes counter after saving' do
+      expect { like.save }.to change { post.reload.likes_counter }.by(1)
+    end
+  end
 end
```

Comment:
The tests are not passing as shown below. Kindly ensure that all the test are passing before resubmitting for review. You can consider adding screenshots of the test in the pull request as evidence. :+1: 

![Screenshot from 2023-03-18 15-27-55](https://user-images.githubusercontent.com/65233860/226115113-a90d95ea-f93a-4a5a-97d0-33be2af4e8d7.png)